docs/docs/guides/labs/dg/deploying-dg.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Deploying to Dagster+'
+sidebar_position: 350
+---
+
+import DgComponentsPreview from '@site/docs/partials/\_DgComponentsPreview.md';
+
+<DgComponentsPreview />
+
+This guide covers how to deploy a `dg`-compatible project to Dagster+.
+
+## Prerequisites
+
```

Comment:
I would add a line here and reformat slightly so you end up with a parallel list:

```
Before you begin, make sure you have the following:

* The `dg` command-line interface installed on your system
* A `dg`-compatible project...
* A Dagster+ account...
* If you're using Dagster+ Hybrid...
```